README.md

```diff
@@ -37,7 +39,7 @@ Based on the current functionality you should consider using Kubeflow if:
 
   * You want to train/serve TensorFlow models in different environments (e.g. local, on prem, and cloud)
   * You want to use Jupyter notebooks to manage TensorFlow training jobs
-       * kubeflow is particularly helpful if you want to launch training jobs that use more resources (more nodes or more GPUs) than your notebook.
+  * You want to launch training jobs that use additional resources (more nodes or more GPUs)
```

Comment:
I think the original meant "notebook" in the sense of "laptop computer" and not in the sense of "Jupyter notebook."  I agree that it was confusing before!

Reply:
Lol. That makes more sense @willb. Happy to add laptop computer back if desired. Working on Jupyter, I always associate notebook with Jupyter notebook ;-)

Reply:
@willingc I only thought of it because of the number of times I've said something like "most data scientists like to do their daily work in their notebooks on their notebooks" ;-)

Reply:
what about `You want to launch training jobs that use resources -- such as additional CPUs or GPUs -- that aren't available on your personal computer`?

Reply:
Great suggestion. Added.